kiss

```diff
@@ -986,10 +986,15 @@ pkg_remove_files() {
     # installation and package removal. Combining the removals in these two
     # functions allows us to stop duplicating code.
     while read -r file; do
-        case $file in /etc/*)
-            printf 'Skipping removal of %s\n' "$file"
-            continue;
-        esac
+        case $file in /etc/?*[!/])
```

Comment:
just as a test i ran this, and found it always returns true

```sh
case /etc/file/does_not/exist in /etc/?*[!/])
  echo "it does exist"
esac
```

Reply:
Yes. That is intended behavior.

Reply:
This just matches files and /not/ directories or `/etc/`

Reply:
ahh so this is kinda equivalent to `echo "/etc/file/does_not_exist" | grep -q '^/etc/.*'` - it is not checking that the file exists, just that the string starts with /etc/ and does not end with /.